src/components/Badge/BadgeReport.jsx

```diff
@@ -319,14 +320,19 @@ const BadgeReport = (props) => {
                   []
                 )}
                 <td style={{ textAlign: 'center' }}>
-                  <Input
-                    type="checkbox"
-                    id={value.badge._id}
-                    checked={value.featured}
-                    onChange={(e) => {
-                      featuredChange(value, index, e);
-                    }}
-                  />
+                  <FormGroup check inline >
```

Comment:
So the issue here was that reactstrap was placing a placing an absolute positioning on the checkbox input for some reason. So as noted in the comment, this can also be fixed by overriding the position style within the input: `style={{ position: 'static' }} `
Though, that felt a bit hacky to me, so I opted to use something in reactstrap. Though this does not look at clean.